fxgl/src/main/kotlin/com/almasb/fxgl/dsl/components/view/GenericBarViewComponent.kt

```diff
@@ -52,4 +52,5 @@ open class GenericBarViewComponent @JvmOverloads constructor(
     fun valueProperty(): DoubleProperty = bar.currentValueProperty()
     fun maxValueProperty(): DoubleProperty = bar.maxValueProperty()
 
+    override fun isComponentInjectionRequired(): Boolean = false
```

Comment:
Same comment here, this is an open class, so a subclass may still require injection

Reply:
Fixed